core/shared/src/main/scala/scalaz/zio/internal/StackBool.scala

```diff
@@ -27,30 +27,32 @@ private[zio] final class StackBool private () {
   private[this] var _size = 0L
 
   final def getOrElse(index: Int, b: Boolean): Boolean = {
-    var j   = index.toLong
+    val i0  = _size & 63L
+    val i   = ((64L - i0) + index) & 63L
+    var ib  = ((64L - i0) + index) / 64L
```

Comment:
```suggestion
    var ib  = ((64L - i0) + index) >> 6
```

Reply:
Thanks for the input, just out of curiosity. Why do you think it's better?

Reply:
I think bitwise shift is much faster than division. You can as well avoid computing `(64L - i0) + index` twice 😃